### ISSUE TITLE
Support Backchannel logout parameters

### ISSUE THREAD

**Issue by kopernic-pl** [auth0/go-auth0#373]
### Checklist

- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

I see that sdk does not support parameters of backchannel logout (apart of URLs)

![image](https://github.com/auth0/go-auth0/assets/3717613/a482398c-bc58-4a5e-b7e7-6da24a425ec7)

This change is needed to get the terraform-auth0 to support this, which in turn is needed for Auth0 pulumi provider.

### Describe the ideal solution

Ideally - 

all the parameters of Auth0 management API related to backchannel logout

```
  "oidc_logout": {
    "backchannel_logout_urls": [
      "string"
    ],
    "backchannel_logout_initiators": {
      "mode": "custom",
      "selected_initiators": [
        "rp-logout"
      ]
    }
  },
```

**Comment by developerkunal** [comment]
HI @kopernic-pl , The support for this feature has been added, but we haven't released it yet. The PR has been merged.

**Comment by developerkunal** [comment]
Hi @kopernic-pl ,

The feature you requested has been released in version 1.5.0. Please take a look and let us know if you encounter any issues.
